tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -18,4 +18,12 @@
 "Microsoft.Azure.Commands.Management.CognitiveServices.dll","Microsoft.Azure.Commands.Management.CognitiveServices.SetAzureCognitiveServicesAccountCommand","Set-AzureRmCognitiveServicesAccount","0","3000","The type of property 'AccountType' of type 'PSCognitiveServicesAccount' has changed from 'System.Nullable`1[Microsoft.Azure.Management.CognitiveServices.Models.Kind]' to 'System.String'.","Change the type of property 'AccountType' back to 'System.Nullable`1[Microsoft.Azure.Management.CognitiveServices.Models.Kind]'."
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Chef.SetAzureVMChefExtensionCommand","Set-AzureRmVMChefExtension","0","2050","A validate set has been added for parameter 'Daemon' for cmdlet 'Set-AzureRmVMChefExtension'.","Remove the validate set from parameter 'Daemon'."
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Chef.SetAzureVMChefExtensionCommand","Set-AzureRmVMChefExtension","0","2090","The ValidateNotNullOrEmpty attribute has been added to parameter 'Linux' for cmdlet 'Set-AzureRmVMChefExtension'.","Remove the ValidateNotNullOrEmpty attribute from parameter 'Linux'."
-"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Chef.SetAzureVMChefExtensionCommand","Set-AzureRmVMChefExtension","0","2090","The ValidateNotNullOrEmpty attribute has been added to parameter 'Windows' for cmdlet 'Set-AzureRmVMChefExtension'.","Remove the ValidateNotNullOrEmpty attribute from parameter 'Windows'."
\ No newline at end of file
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Chef.SetAzureVMChefExtensionCommand","Set-AzureRmVMChefExtension","0","2090","The ValidateNotNullOrEmpty attribute has been added to parameter 'Windows' for cmdlet 'Set-AzureRmVMChefExtension'.","Remove the ValidateNotNullOrEmpty attribute from parameter 'Windows'."
```

Comment:
Instead of this, add the alias 'Name' to each of the ExpressRouteCircuitName parameters.